README.md

```diff
@@ -155,9 +155,31 @@ spark.sql("select count(*) from lineitem").show
 spark.sql("select ti_version()").show
 ```
 
-## TiDB Data Source API
+## Write Data To TiDB using TiDB Connector
+TiSpark natively supports writing data to TiKV using Spark Data Source API and guarantees ACID.
```

Comment:
s/TiSpark natively supports writing data to TiKV using Spark Data Source API and guarantees ACID.
Write/TiSpark natively supports writing data to TiKV using Spark Data Source API. ACID is also guaranteed.

Reply:
I think `and guarantees ACID` is better... You might also change it to `and ACID guarantees`